Fix-Website/app/app.js

```diff
@@ -4,28 +4,26 @@
     app.directive("contentTable", function(){
         return {
             restrict: 'E',
-            templateUrl: "Templates/content-table.html",
+            templateUrl: "Templates/content-table.txt",
             controller: function($scope) {
```

Comment:
We should definitely define the controllers elsewhere, and not included them here. The Google style guide for JavaScript breaks lines at 80-chars, and already 20+ is wasted on just indentation for the app.directive's parameter, which is a lambda, which returns a JavaScript object literall, which has a field named controller, which has value of a lambda expression, which inside of the anonymous function's body has a method named clickProduct, which has an if statement, etc.